pkg/detectors/deepseek/deepseek.go

```diff
@@ -0,0 +1,115 @@
+package deepseek
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	regexp "github.com/wasilibs/go-re2"
+	"io"
+	"net/http"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+
+	keyPat = regexp.MustCompile(`\b(sk-[a-z0-9]{32})\b`)
+)
+
+// Keywords are used for efficiently pre-filtering chunks.
+// Use identifiers in the secret preferably, or the provider name.
+func (s Scanner) Keywords() []string {
+	return []string{"sk-"}
```

Comment:
@bunnyanon `sk-` is not a sufficient keyword. `deepseek` is necessary for filtering out fps.